packages/trader/src/App/Components/Elements/ContractAudit/contract-history.tsx

```diff
@@ -26,11 +29,11 @@ const ContractHistory = ({ currency, history = [] }) => {
                         key={key}
                         id={`dt_history_label_${key}`}
                         label={item.display_name}
-                        timestamp={+item.order_date}
+                        timestamp={Number(item?.order_date)}
                         value={
-                            Math.abs(+item.order_amount) !== 0 ? (
+                            Math.abs(Number(item.order_amount)) !== 0 ? (
                                 <React.Fragment>
-                                    {+item.order_amount < 0 && <strong>-</strong>}
+                                    {Number(item.order_amount) < 0 && <strong>-</strong>}
```

Comment:
Why do we need this change? are you expecting item.order_amount to a different value than number or a string?

Reply:
wrapping it with `Number` function gets rid of `item.order_amount` being `undefined` error